roles/edpm_nova/defaults/main.yml

```diff
@@ -28,8 +28,8 @@ edpm_nova_image_download_retries: "{{ edpm_download_retries | default(5) }}"
 
 # Note that the src dir is in the AEE container but the
 # dest dir is on the target host
-edpm_nova_config_src: /var/lib/openstack/configs
-edpm_nova_config_dest: /var/lib/openstack/config/nova
+edpm_nova_config_src: /var/lib/openstack/configs/{{ edpm_nova_service_name }}
+edpm_nova_config_dest: /var/lib/openstack/{{ edpm_nova_service_name }}
```

Comment:
changing this path also affects sos reports https://github.com/search?q=repo%3Asosreport%2Fsos%20%2Fvar%2Flib%2Fopenstack&type=code